.editorconfig

```diff
@@ -108,3 +108,7 @@ charset = unset
 [lib/tests/*.plist]
 indent_style = tab
 insert_final_newline = unset
+
+[pkgs/kde/generated/**]
+insert_final_newline = unset
+end_of_line = unset
```

Comment:
We should really set those otherwise the diff will probably sometimes be broken.

Reply:
It's just autogenerated JSON stuff, it should be consistent (LF line endings, no final newline).